test/valid-data/class-single/main.ts

```diff
@@ -17,7 +17,7 @@ export class MyObject {
     readonly readonlyProp: string;
 
     // Constructors must be ignored
-    public constructor(protected a: number, private b: number, c: number, public propC: number,
+    public constructor(protected a: number, private b: number, c: number, public propC = 42,
```

Comment:
You could add a comment to help identify that the `= 42` is used to test inferrable types.